vancouver-tree-height-geo.Rmd

```diff
@@ -13,17 +13,23 @@ knitr::opts_chunk$set(echo = TRUE)
 
 ## Summary
 
-Our group was interested in answering the question, "do tree heights vary significantly from neighbourhood to neighbourhood in Vancouver?"
+Our group was interested in answering the question, "**Do tree heights vary significantly from neighborhood to neighborhood in Vancouver?**"
+
+We aim to analyze the relationship between tree height distribution and neighborhoods. Specifically, we focus on tree height ranges and their counts in various neighborhoods to determine if tree height is influenced by location.
 
 ## Introduction
 
-The City of Vancouver Open Data Portal has .... all this data
+Urban trees are essential to creating livable cities, offering ecological, aesthetic, and health benefits. They improve air quality, provide shade, support biodiversity, and enhance the overall urban environment. In Vancouver, street trees play a vital role in the city’s commitment to sustainability. However, the characteristics of these trees, such as their heights, can vary significantly across neighborhoods due to factors like local urban planning, soil quality, and maintenance practices. Understanding these patterns is key to equitable urban forestry management and informed decision-making.
+
+This project explores the relationship between **tree height distribution** and **neighborhoods** in Vancouver. Using data from the [City of Vancouver Open Data Portal](https://opendata.vancouver.ca/explore/dataset/street-trees/information/?disjunctive.species_name&disjunctive.common_name&disjunctive.on_street&disjunctive.neighbourhood_name), we seek to answer the question: **Do tree heights vary significantly between neighborhoods?**
```

Comment:
The final clause in the last sentence, `we seek to answer the question: **Do tree heights vary significantly between neighborhoods?**`, is repetitive since we already said this in the first sentence of the paragraph. We can probably delete the last clause of the last sentence (everything after the comma), but note that the sentence will need to be reworded, something like `To answer this question, we used data from the [...]`

Reply:
Changes done! Thanks for the suggestions